hpx/runtime/actions/continuation.hpp

```diff
@@ -172,10 +172,17 @@ namespace hpx { namespace actions
         }
         HPX_SERIALIZATION_POLYMORPHIC_ABSTRACT(continuation);
 
+#if defined(HPX_HAVE_COMPONENT_GET_GID_COMPATIBILITY)
         naming::id_type const& get_gid() const
         {
             return gid_;
         }
+#endif
+
+        naming::id_type const& get_id() const
+        {
+            return gid_;
+        }
```

Comment:
Why should it? This was renamed for consistency reasons only. The user will never see or call this function directly anyways.